### PR TITLE
Update version to match Podman

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@jwhonce @4383 @baude @saschagrunert PTAL

**Comment by rhatdan** [comment]
Want to add a version to match podman and then cut a tag/release.

**Comment by rhatdan** [comment]
@toabctl PTAL

File: setup.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 setup(
     name='podman',
-    version=os.environ.get('PODMAN_VERSION', '0.0.0'),
+    version=os.environ.get('PODMAN_VERSION', '1.4.3'),
```
Comment (4383) [inline setup.py]: It's not an issue to merge these changes but they will be removed if we merge my changes.

Also I've some comments:
1. My proposed changes on #25 would avoid to do this and would manage this by using `git tag`.

2. I don't know in which cases the `PODMAN_VERSION` env var can exist on the system, I suppose when podman is installed, so in this case if we install a different version of `python-podman` not the really the same than `podman` you need to assume that they can be incompatible or things like that... I'm sure it's a good things to retrieve the python client version from the podman version... 
Also how to manage the client version if we want to install from pypi? If you choose to use the env var to determine the python package version, and if you build it locally and the then publish it on pypi you need to take care to have the right podman installed version.

I hope my comment can help you.

Do not hesitate to ask things if needed.

Comment (rhatdan) [reply setup.py]: I am willing to entertain other ideas.  The version of Podman that we build with python-podman should be pretty close.  I would expect that podman-python 1.4.3 should work with podman 1.4.3 or later versions.  Might work with older versions.

**Comment by rhatdan** [comment]
Closing since I took @4383 Version.

**Pull request closed without merging**